### PR TITLE
[search] Get rid of T-prefix.

### PR DESCRIPTION
**Pull request by ygorshenin**

### CODE REVIEW

File: indexer/indexer_tests/trie_test.cpp [hunk 1]
```diff
@@ -37,76 +33,78 @@ struct ChildNodeInfo
...
-    while (src.Size() > 0)
-      m_values.emplace_back(ReadPrimitiveFromSource<TValue>(src));
+    while (source.Size() > 0)
+      m_values.emplace_back(ReadPrimitiveFromSource<Value>(source));
   }
 
-  template <typename TF>
-  void ForEach(TF && f) const
+  template <typename ToDo>
+  void ForEach(ToDo && toDo) const
```
Comment (mpimenov) [inline indexer/indexer_tests/trie_test.cpp]: Fn?

File: indexer/trie_builder.hpp [hunk 2]
```diff
@@ -74,7 +74,7 @@ void WriteNode(TSink & sink, TSerializer const & serializer, TrieChar baseChar,
   if (childCount >= 63)
     WriteVarUint(sink, childCount);
   valueList.Serialize(sink, serializer);
-  for (TChildIter it = begChild; it != endChild; /*++it*/)
+  for (ChildIter it = begChild; it != endChild; /*++it*/)
```
Comment (mpimenov) [inline indexer/trie_builder.hpp]: The comment is superfluous.

File: indexer/trie_builder.hpp [hunk 3]
```diff
@@ -131,11 +131,11 @@ struct ChildInfo
   size_t GetEdgeSize() const { return m_edge.size(); }
 };
 
-template <typename TValueList>
+template <typename ValueList>
 struct NodeInfo
 {
-  uint64_t m_begPos;
-  TrieChar m_char;
+  uint64_t m_begPos = 0;
```
Comment (mpimenov) [inline indexer/trie_builder.hpp]: Consider moving it to the bottom while you're here.

File: indexer/trie_builder.hpp (hunk 2 above)
Comment (ygorshenin) [reply indexer/trie_builder.hpp]: Done

File: indexer/trie_builder.hpp (hunk 3 above)
Comment (ygorshenin) [reply indexer/trie_builder.hpp]: Done

File: indexer/indexer_tests/trie_test.cpp (hunk 1 above)
Comment (ygorshenin) [reply indexer/indexer_tests/trie_test.cpp]: I'd prefer to keep the name as is.

**Comment by ygorshenin** [comment]
PTAL

**Pull request merged**